vowpalwabbit/parse_args.cc

```diff
@@ -661,8 +660,6 @@ void parse_feature_tweaks(options_i& options, vw& all, std::vector<std::string>&
                .help("Don't remove interactions with duplicate combinations of namespaces. For ex. this is a "
                      "duplicate: '-q ab -q ba' and a lot more in '-q ::'."))
       .add(make_option("quadratic", quadratics).short_name("q").keep().help("Create and use quadratic features"))
-      // TODO this option is unused - remove?
-      .add(make_option("q:", q_colon).help("DEPRECATED ':' corresponds to a wildcard for all printable characters"))
```

Comment:
Have we had a major release with this deprecated? If no, I would prefer it if we did not remove it outright, yet.

Reply:
There is no way to actually call it, since `-q:` gets interpreted as `-q :`. If you can find a way to hit this code then I will hold off on removing it

Reply:
Nevermind, it is possible to access it. Will close this PR.